frameworks/Elixir/phoenix/setup.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+# sed -i 's|db_host: "localhost",|db_host: "${DBHOST}",|g' config/config.exs
```

Comment:
Perhaps delete the line if it's unnecessary if you're making more changes to the PR